core/Dockerfile

```diff
@@ -23,5 +22,5 @@ EXPOSE 8001 8024
 
 ENV MAILMAN_CONFIG_FILE /etc/mailman.cfg
 
-ENTRYPOINT ["/opt/run.sh"]
-CMD ["/usr/local/bin/master"]
+ENTRYPOINT ["docker-entrypoint.sh"]
+CMD ["master"]
```

Comment:
It's implicitly set and used/exported.
Reference: https://github.com/moby/moby/blob/f154588/pkg/system/path.go#L5